parlai/core/dict.py

```diff
@@ -101,6 +102,7 @@ class DictionaryAgent(Agent):
     default_tok = 're'
     default_lower = False
     default_textfields = 'text,labels'
+    default_cachesize = 1048576
```

Comment:
why this?

Reply:
Cache is most powerful if it's a power of 2. I'll add a comment.